Code/Functions/Public/format-HTML.ps1

```diff
@@ -0,0 +1,121 @@
+function format-HTML {
```

Comment:
I think we discussed that this function was actually not needed since this functionality is provided by all the code editors and the browsers themselves.

Would you please mind removing this one from the PR?